### PR TITLE
[rxjava] Initial Integration

### PR DESCRIPTION
**Pull request by aschaich**

### CODE REVIEW

File: projects/rxjava/src/main/java/ossfuzz/IntegerObserver.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
+package ossfuzz;
```
Comment (Navidem) [inline projects/rxjava/src/main/java/ossfuzz/IntegerObserver.java:1]: Missing Copyright header?

File: projects/rxjava/src/main/java/ossfuzz/ObservableFuzzer.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
+package ossfuzz;
```
Comment (Navidem) [inline projects/rxjava/src/main/java/ossfuzz/ObservableFuzzer.java:1]: same here

File: projects/rxjava/src/main/java/ossfuzz/IntegerObserver.java (hunk 1 above)
Comment (aschaich) [reply projects/rxjava/src/main/java/ossfuzz/IntegerObserver.java]: Added our licence headers

**Review by jonathanmetzman — APPROVED** [review]
LGTM

**Pull request merged**